db/migrate/20200831115416_add_type_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddTypeToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :type, :string
```

Comment:
Once you use enum, column name won't be `type` anymore as it is specific to STI in Rails